book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,173 @@
+  Or perhaps a foundation or other non-profit/academic organisation with robust funding.
+- What are the data export options and formats?
+
+Open solutions provide you with the assurance that if you do the appropriate preparatory work you should be able to access all of your data in it's native form by running the ELN application in a virutual machine or similar reproducible computational environment in the future should you need to.
+Even if the tools are no longer maintained and in a state that can be used in production.
+They can still be used to read the data and interact with it in (mostly) the same way.
+The data will also likely be stored in an open format from which it can relatively easily be extracted and ported to a new format.
+This may still be a lot of work but it is much more attainable than reverse engeering a proprietary format for which there is no documentation or worse still dealing with encrypted files.
+
+Most proprietary or open tools will permit you to export your data in one format or another, however the quality of this export is critical to scrutinise.
+The ability to save all your lab books as PDFs is fine but if it's the only option you may loose a lot of metadata, and the ability to import your data into a new solution. 
+Migration may cost you much of your data's structure.
+
+Proprietary solutions are incentivised to attempt to lock-in their customers so that the cost of switching to a competitor is high so be especially wary of poor export options.
+Note that this tends to change over time. 
+New providers are focused on user aquisition so data portability tends to be good at first but decline if a firm has to shift from a growth phase to a steady state that is sustainably profitable.
+Mendeley for example began [encrypting their local database](https://www.zotero.org/support/kb/mendeley_import#mendeley_database_encryption) making it impossible to migrate your local library to different reference managers without going through their online library feature.
+This occured after they had reached a high degree of market penetration and been aquired by Elsevier.
+
+Many open tools on the other hand have very robust exort and import options, as they are not subject to this incentives trap, though this is far from garunteed.
+This might include a complete database export that would let you seemlessly migrate to a different deployment of the tool. 
+Or an archive of files organised by a for example a JSON file representation of your notes and files that retain most or all of the metadata and organisational information of your ELN.
+There remains the challenge when migrating of finding a tool that can import what your old tool exported, and retain most of all of its structure.
+This is where the development of standards for ELN formats could very valuable in the long run.
+
+This recent review {cite:ps}`higgins2022elns` provides a good overview of considerations when adopting an ELN solution. 
```

Comment:
I feel it is better to start this section as a consideration of whether hosting a solution by a third party is an option, rather than starting with this resource. Instead I think it is better to put all the 'what ELN solution to pick' in one bullet point paragraph below.